src/views/Home/components/Question/questionData.js

```diff
@@ -34,14 +34,20 @@ export async function getQuestions() {
   
   await getUserData();
   let backendQuestions = await wellness.getQuestion();
-
-  let phoneNumber = `(${user.MobilePhone.substring(0, 3)}) ${user.MobilePhone.substring(
-    3,
-    6,
-  )}-${user.MobilePhone.substring(6)}`;
-
-  //comment below disables the eslint warning so we can execute dynamic string parsing without warnings
   
+  let phoneNumber;
+  
+  if(user.MobilePhone === undefined){
+     phoneNumber = "your phone number";
```

Comment:
I think that if there's no mobile phone, then we should try home phone, and if theres no home phone then maybe this fix or email works.

Reply:
fixed it!